test/jose-jws-test.html

```diff
@@ -32,18 +33,18 @@
                             b64uxp.test(signature.payload) &&
                             b64uxp.test(signature.signature);
                 });
-            }), true, "got right JSON serialization format");
+            }), true, "got right JSON serialization format 1");
 
             assert.willEqual(ads_promise.then(function () {
                 return signer.sign(plain_text, null, {}).then(function (signature) {
                     var xp = /([a-z0-9_-]+\.){2}[a-z0-9_-]+/i;
                     return xp.test(signature.CompactSerialize());
                 });
-            }), true, "got correct compact serialization format");
+            }), true, "got correct compact serialization format 2");
```

Comment:
I think the '2' is a remnant of digging into the problem. Should be removed.

Reply:
also Fixed.